docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -117,7 +115,7 @@ If you wanted to test out the tool, you can run it against :doc:`DAML Sandbox
 
      $ java -jar ledger-api-test-tool.jar --extract
      $ daml sandbox -- *.dar
-     $ java -jar ledger-api-test-tool.jar
+     $ java -jar ledger-api-test-tool.jar localhost:6865 --all-tests
```

Comment:
I would recommend non changing the meaning of this line at this time. Furthermore, it's advisable to not run all tests at the same time but to run those who are quite aggressive in terms of resource usage or that change the global state of the ledger separately. Given what we're trying to explain here I believe that leaving in just the semantics is perfectly fine.

Reply:
this line doesn't work without <host>:<port>
i can leave off `--all-tests` for now though